anchore_engine/services/catalog/swagger/swagger.yaml

```diff
@@ -352,6 +352,36 @@ paths:
           schema:
             type: object
       x-swagger-router-controller: "anchore_engine.services.catalog.api.controllers.default_controller"
+  /images/{image_digest}/content:
```

Comment:
I considered replacing the original API but didn't want to combine it with this PR because of the regression potential. The existing image-content-by-type API is in the hot path, so a follow up ticket would be my recommendation.